lapka/lib/components/basic/imageButton.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter_svg/svg.dart';
```

Comment:
To jest dobra podstawa do buttona 'adopt button', wystarczy trochę rozszerzyć funkcjonalność.